src/Molecules/List/ListItem/ListItem.jsx

```diff
@@ -1,12 +0,0 @@
-import styled from 'styled-components'
```

Comment:
why ?
When I click on the items now,  I have the ripple. I don't want that
![](https://media1.giphy.com/media/9Y5BbDSkSTiY8/giphy.gif)

Reply:
Can you show me where on the documentation you find the ripples options I removed?